no_look_pass_robot/vec2d[] laser_data
float32 rangeMax
